docs/source/generic.md

```diff
@@ -29,6 +29,9 @@ Then make kubectl aware of this local installation like this:
 eval $(minikube docker-env)
 ```
 
+## Download Scylla Operator
+In this guide you will be using the examples and manifests from [Scylla Operator repository](https://github.com/scylladb/scylla-operator), so start off by cloning it to your local machine.
```

Comment:
Other paragraphs suggests what command to use, this one is missing one

Reply:
done

Reply:
adding `cd scylla-operator`  would allow copy-pasting every command sequentially